js/validation.js

```diff
@@ -0,0 +1,65 @@
+import {formElement, HOUSING_MIN_PRICE} from './data.js';
+const adTitleInput = formElement.querySelector('#title');
+const typeSelect = formElement.querySelector('#type');
+const priceInput = formElement.querySelector('#price');
+const countRoomsSelect = formElement.querySelector('#room_number');
+const countGuestsSelect = formElement.querySelector('#capacity');
+
+const valuePriceCheck = () => {
+  priceInput.min = HOUSING_MIN_PRICE[typeSelect.value];
+  (Number(priceInput.value) < Number(priceInput.min)) ? priceInput.setCustomValidity(`Минимальная цена за ночь ${  priceInput.min}`) : priceInput.setCustomValidity('');
```

Comment:
Еще раз, скобки вокруг сравнений не нужны
```suggestion
  Number(priceInput.value) < Number(priceInput.min) ? priceInput.setCustomValidity(`Минимальная цена за ночь ${  priceInput.min}`) : priceInput.setCustomValidity('');
```
Также вместо Number можно пользоваться сокращенной записью и поставить плюс
```suggestion
  +priceInput.value < +priceInput.min ? priceInput.setCustomValidity(`Минимальная цена за ночь ${  priceInput.min}`) : priceInput.setCustomValidity('');
```

Можно заметить, что в обеих ветках тернарика происходит обращение к setCustomValidity(), от дублирования можно избавиться, если перенесем тернарник внутрь setCustomValidity
```suggestion
  priceInput.setCustomValidity(+priceInput.value < +priceInput.min ? `Минимальная цена за ночь ${  priceInput.min}` : '')
```
---
Это не замечание к исправлению, а скорее заметка по поводу того, как должен идти процесс оптимизации и рефакторинга

Reply:
Поля **#title** и **#price** не требуют сложной **js** валидации, т.к. хорошо валидируются средствами браузера.

- Для **title** достаточно задать атрибуты в верстке.
- При изменении поля **type** нужно изменять атрибуты **placeholder** и **min** у поля **price**